### PR TITLE
fix: include tests in type check

### PR DESCRIPTION
**Pull request by fa93hws**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
Tests should be included in typecheck

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ components/**/*.jsx
 !components/*/demo/*
 !.*.js
 # Docs templates
+*.md
```
Comment (afc163) [inline .eslintignore:7]: Why ignore md from eslint?

Comment (fa93hws) [reply .eslintignore:7]: Are we linting markdown file with eslint?
`components/table/index.en-US.md` are full of red lines..

If it's expected to lint markdown file with eslint, I will try to find some other workaround in another PR

Comment (afc163) [reply .eslintignore:7]: `npm run lint:demo`

Comment (fa93hws) [reply .eslintignore]: Cool, I will remove this ignore here.
And I do have some red lines in `components/table/demo/ajax.md` as well...
Will take a look when I have time.

File: .eslintrc.js [hunk 2]
```diff
@@ -1,7 +1,3 @@
-const commonGlobals = {
-  gtag: true,
-};
-
```
Comment (fa93hws) [inline .eslintrc.js:4]: Actually the `overrides` will handle it for usd

**Comment by afc163** [comment]
image

**Comment by fa93hws** [comment]
> image

Hmmm interesting.. I did add few lines so that the typescript is happy about the types and there was no test for this file before..
But after having a comprehensive read on `domHook.ts`, I have a feeling that it's not necessary at all... Let me have  a try on removing it..

**Pull request merged**